packages/extension/src/providers/solana/libs/api.ts

```
@@ -23,36 +23,48 @@ class API implements ProviderAPIInterface {
+
+    if (!tx) {
+      // Transaction hasn't been picked up by the node
+      // (maybe it's too soon, or maybe node is behind, or maybe it's been dropped)
+      return null;
+    }
+
+    const retVal: SOLRawInfo = {
+      blockNumber: tx.slot,
+      timestamp: tx.blockTime,
+      transactionHash: hash,
+      status: tx.meta?.err ? false : true,
```

Biome

[error] Unnecessary use of boolean literals in conditional expression.

Simplify your code by directly assigning the result without using a ternary operator.
If your goal is negation, you may use the logical NOT (!) or double NOT (!!) operator for clearer and concise code.
 Check for more details about NOT operator.
Unsafe fix: Remove the conditional expression with


(lint/complexity/noUselessTernary)